models/migrations/v16.go

```diff
@@ -0,0 +1,21 @@
+package migrations
+
+import "github.com/go-xorm/xorm"
+
+func dropIssueUserColumns(x *xorm.Engine) error {
+	// different SQL databases have different syntax for dropping multiple
+	// columns, so drop columns one at a time
+	if _, err := x.Exec("ALTER TABLE issue_user DROP COLUMN repo_id"); err != nil {
+		return err
+	}
+	if _, err := x.Exec("ALTER TABLE issue_user DROP COLUMN milestone_id"); err != nil {
+		return err
+	}
+	if _, err := x.Exec("ALTER TABLE issue_user DROP COLUMN is_poster"); err != nil {
+		return err
+	}
+	if _, err := x.Exec("ALTER TABLE issue_user DROP COLUMN is_closed"); err != nil {
```

Comment:
On Gogs, the convention was never delete columns in the DB to prevent break anything, specially if user tries to rollback to a previous version of Gogs/Gitea.

Should we keep this convention?